src/olympia/addons/cron.py

```diff
@@ -75,22 +75,15 @@ def update_addon_download_totals():
     if not waffle.switch_is_active('local-statistics-processing'):
         return False
 
-    cursor = connections[multidb.get_slave()].cursor()
-    # We need to use SQL for this until
```

Comment:
Looking at the `EXPLAIN` of this query in prod it wasn't necessary to force the index, MySQL was doing it on its own. Furthermore, rewriting the query using the ORM, starting from `addons` rather than `download_counts`, reduces the number of rows examined by a lot, since we can eliminate all deleted add-ons first.

Reply:
By the way, the `USE KEY` was the reason why it wasn't unit tested, since that custom index is never added in tests. We'll need Django 1.11 `indexes` syntax if we want to add indexes with custom names.